app/lib/secrets_manager.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'aws-sdk-secretsmanager'
+
+module SecretsManager
+  class << self
+    # @param secret_id [String] - The secret ID to get the value for
+    # @return [String] - The value of the stored secret
+    def get_value(secret_id)
+      JSON.parse(secrets_manager.get_secret_value(
+        secret_id: secret_name(secret_id)
+      ).secret_string)
+    rescue StandardError => e
+      logger.warn("Failed to get secret value for #{secret_name(secret_id)}!\n#{e}")
```

Comment:
@subbiahf22 I changed this on the champaign's PR to just return nil.
You can leave it, but I think `logger` is not defined, if so, it should be `Rails.logger`
Could you please verify how we are calling the logger on this codebase?